cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-verify-item-status.cy.js

```diff
@@ -0,0 +1,71 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import FileManager from '../../../support/utils/fileManager';
+
+describe('bulk-edit', () => {
+  describe('in-app approach', () => {
+    let userData = {};
+    const instanceData = {
+      instanceName: `C353652 testBulkEdit_${getRandomPostfix()}`,
+      itemBarcode: getRandomPostfix(),
+    };
+    let fileContent = '';
+    const itemBarcodesFileName = `C353652 itemBarcodes_${getRandomPostfix()}.csv`;
+
+    before('Create test data', () => {
+      cy.createTempUser([
+        Permissions.bulkEditEdit.gui,
+        Permissions.uiInventoryViewCreateEditDeleteItems.gui,
+        Permissions.bulkEditView.gui,
+      ]).then((userProperties) => {
+        userData = userProperties;
+        cy.login(userData.username, userData.password, {
+          path: TopMenu.bulkEditPath,
+          waiter: BulkEditSearchPane.waitLoading,
+        });
+        InventoryInstances.createInstanceViaApi(
+          instanceData.instanceName,
+          instanceData.itemBarcode,
+        );
+        fileContent += instanceData.itemBarcode;
+        FileManager.createFile(`cypress/fixtures/${itemBarcodesFileName}`, fileContent);
+      });
+    });
+
+    after('Delete test data', () => {
+      Users.deleteViaApi(userData.userId);
+      FileManager.deleteFile(`cypress/fixtures/${itemBarcodesFileName}`);
```

Comment:
you should delete created instance as well, see `deleteInstanceAndHoldingRecordAndAllItemsViaApi`